packages/alfa-aria/test/get-name.spec.tsx

```diff
@@ -35,7 +35,7 @@ test("getName() computes the text alternative of a button with text within a spa
 test("getName() ignores non-visible nodes", (t) => {
   const button = Element.fromElement(
     <button>
-      Button <span style="display: none">Hidden</span>
+      Button <span style={{ display: "none" }}>Hidden</span>
```

Comment:
I am a bit concerned by the need for double curly brackets. Notably because that make copy/pasting HTML a bit more difficult.

Reply:
That's how JSX works though. JSX isn't HTML and that's both good and sort of bad. In this case, passing a property-value record greatly simplifies that internal code and makes it easier for consumers to define styles as things aren't stringly typed anymore. In fact, we can improve the typing further by strongly typing the individual properties, though that would be an improvement for the future.

Reply:
Also relevant: https://reactjs.org/docs/dom-elements.html#style

Reply:
True. And I have being doing `const style={…}; <div style={style}>` quite a few times… 🤷